tests/features.py

```diff
@@ -0,0 +1,41 @@
+from flask import current_app
+from mock import patch
+
+
+def enable_feature(name, value=True):
```

Comment:
I think there was some reason why we couldn't go with 
```python
def enable_flag(name, value=True):
    PATCH = {'FEATURE_FLAGS_{}'.format(name.upper()): value}
    return patch.dict(current_app.config, PATCH)
```
but I don't think it's captured in this PR. Do you remember what it was?